webextension/background.js

```diff
@@ -141,10 +146,15 @@ const TabState = (function() {
       this._reportSubmitPromise = new Promise(async (resolve, reject) => {
         const report = this._report;
         const { incognito, url } = await browser.tabs.get(this._tabId);
-        if (incognito) {
+        if (incognito !== undefined) {
           report.incognito = incognito;
         }
-        report.url = url;
+        if (report.includeURL !== undefined) {
+          if (report.includeURL) {
```

Comment:
What's the purpose of this double `if (report.includeURL)`? Are you trying to check if it's not undefined and not false? Or similar? Why not just `if (report.includeURL)`?

Reply:
It's just to make sure that a value was actually passed before processing it (and removing it from the final dict). I should probably change it to `"includeURL" in report`, if that's alright?

Reply:
Yeah, that makes sense.